### DISCUSSION TITLE
Breaking change: renamed client-provided key properties for applications and groups

### DISCUSSION THREAD

**Discussion by dkershaw10** [microsoftgraph/msgraph-bicep-types#71] [Announcements]
Hello everyone,

We just rolled out an update, to have consistent naming of client-provided key properties between Bicep and Graph types.  This means that in the `applications` and `groups` bicep types, we renamed the `name` key to `uniqueName`.  You'll see this in the latest Bicep client (v0.25.53).  As part of this change, we've also updated our quick start samples. 

See https://github.com/microsoftgraph/msgraph-bicep-types/releases/tag/v0.1.2-preview.

> **NOTE**: If you have any existing templates, you'll need to update them to reflect this change to the `applications` and `group` types.

We're working on a versioning strategy, together with the dynamic type loading feature that the Bicep team is building, that would allow you to choose the provider package for your template.  @jason-dou will post something about this very soon, to get folks' feedback.

**Comment by jason-dou** [comment]
Versioning strategy is posted here: https://github.com/microsoftgraph/msgraph-bicep-types/discussions/72
